base/src/main/scala/co/uproot/abandon/Report.scala

```diff
@@ -250,40 +250,72 @@ object Reports {
     }
   }
 
-  def xmlBalanceExport(state: AppState, exportSettings: XmlExportSettings): xml.Node = {
-    <abandon>
-       <balance>
-          { state.accState.mkTree(exportSettings.isAccountMatching).toXML }
-       </balance>
-    </abandon>
+  def xmlBalanceExport(state: AppState, exportSettings: XmlExportSettings, withVersion: Boolean): xml.Node = {
+    if (withVersion) {
```

Comment:
Thanks @hrj. I was away for a couple of days and didn't want to have the changes laying around so I pushed them to the branch. I was definitely looking for a better implementation and your suggestion is great.